redirect.json

```diff
@@ -110,6 +110,20 @@
     "description": "show your devices",
     "introduced": "2021.3"
   },
+  {
+    "redirect": "config_energy",
+    "name": "Energy Configuration",
+    "badge": "Energy Configuration",
+    "description": "show your Energy configuration panel",
+    "introduced": "2021.8"
+  },
+  {
+    "redirect": "energy",
+    "name": "Energy Panel",
```

Comment:
Wonder if we should call it Energy Management instead ?

Reply:
Or just energy. It's already differentiated because the other one is different.

Reply:
Just energy, that way it's also over the configuration entry in the dropdown.

Reply:
Just energy is awesome. It shows: My Energy :)